build/run.js

```diff
@@ -265,7 +261,7 @@ commands.lint.rust = async function () {
 commands['toml-fmt'] = command(`Lint the codebase`)
 commands['toml-fmt'].rust = async function () {
     console.log('Looking for all TOML files.')
-    let files = glob.sync(paths.rust.root + '/**/*.toml', { cwd: paths.root })
+    let files = glob.sync(paths.root + '/**/*.toml', { cwd: paths.root })
```

Comment:
TOML formatting should be checked as part of Lint workflow. Consider creating an issue for this.

Reply:
Added: https://enso.youtrack.cloud/issue/IDE-3226